apps/sms/views/conversation/js/conversation.js

```diff
@@ -575,6 +576,7 @@ var ConversationView = {
     var prevPanel = args.meta.prev;
 
     if (!this.isConversationPanel(threadId, prevPanel)) {
+      // we don't want to render again if it's already rendered for this thread
```

Comment:
nit: ```for this conversation``` and in some comments below ```thread``` ----> ```conversation```, ```composer``` ----> ```new message```. Not that important, but anyway :)